src/platform/user/profile/vet360/util/local-vet360.js

```diff
@@ -198,91 +198,235 @@ export default {
       },
     };
   },
-  addressValidationSuccess() {
-    return {
-      addresses: [
-        {
-          address: {
-            addressLine1: '400 N 65th St',
-            addressType: 'DOMESTIC',
-            city: 'Seattle',
-            countryName: 'USA',
-            countryCodeIso3: 'USA',
-            countyCode: '53033',
-            countyName: 'King',
-            stateCode: 'WA',
-            zipCode: '98103',
-            zipCodeSuffix: '5252',
+  addressValidationSuccessSingleConfirmedSuggestion() {
```

Comment:
All of these changes are just adding a bunch more mock API calls to use when testing locally. And also makes the existing `addressValidationSuccess()` an async function that resolves after 1s to simulate the kind of delay we are seeing in staging.